apps/mintacoin/lib/mintacoin/wallets/schema.ex

```diff
@@ -30,8 +30,8 @@ defmodule Mintacoin.Wallet do
   @spec changeset(wallet :: %__MODULE__{}, changes :: map()) :: Changeset.t()
   def changeset(wallet, changes) do
     wallet
-    |> cast(changes, [:address, :secret, :account_id, :blockchain_id])
-    |> validate_required([:address, :secret, :account_id, :blockchain_id])
+    |> cast(changes, [:address, :encrypted_secret, :account_id, :blockchain_id])
+    |> validate_required([:address, :encrypted_secret, :account_id, :blockchain_id])
     |> foreign_key_constraint(:account_id)
     |> foreign_key_constraint(:blockchain_id)
   end
```

Comment:
It is important to validate the unique constraint from the changeset by adding this:
```suggestion
    |> foreign_key_constraint(:account_id)
    |> foreign_key_constraint(:blockchain_id)
    |> unique_constraint([:account_id, :blockchain_id], name: :account_blockchain_index)
  end
```

Reply:
Thanks!!!